plone/app/z3cform/configure.zcml

```diff
@@ -39,13 +39,6 @@
       class=".templates.RenderContentProvider"
       permission="zope.Public"
       />
-  <browser:page
-      name="ploneform-render-widget"
-      for="z3c.form.interfaces.ISingleCheckBoxWidget"
-      layer="plone.app.z3cform.interfaces.IPloneFormLayer"
-      class=".templates.RenderSingleCheckboxWidget"
-      permission="zope.Public"
-      />
```

Comment:
The point of this view was to make sure that the label is rendered to the right of the checkbox in input mode and the description underneath. I see you made the label go in the right place, but I think with your change the description will now be shown above the checkbox.